### PR TITLE
Adding linting to CI

### PR DESCRIPTION
**Pull request by arnasbr**

### CODE REVIEW

**Comment by arnasbr** [comment]
https://github.com/traveltime-dev/traveltime-benchmarks/issues/34

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.34.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "prettier": "^3.2.5"
```
Comment (arnasbr) [inline package.json:15]: not sure if all these are actually needed, but won't hurt I guess

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Linting
+
+on:
+  push:
```
Comment (danielnaumau) [inline .github/workflows/lint.yml:4]: pr to master maybe?

Comment (arnasbr) [reply .github/workflows/lint.yml:4]: yea can do that too

**Review by danielnaumau — APPROVED** [review]

**Pull request merged**